src/assets/images/icons/UTC--2018-05-29T21-54-53.653Z--4c08f6527dba8bcdfe05b7fdb3b1243f3cfcbbec

```diff
@@ -0,0 +1 @@
+{"version":3,"id":"6203e68b-529d-408a-bd29-0f088382a2de","address":"4c08f6527dba8bcdfe05b7fdb3b1243f3cfcbbec","Crypto":{"ciphertext":"3a0c7327a2b1eb02fa90ce09cf47345637094e6c70666886fbb8ddb3a3e33815","cipherparams":{"iv":"474bdbfd2f4b3c71e53bb3d30af75d61"},"cipher":"aes-128-ctr","kdf":"scrypt","kdfparams":{"dklen":32,"salt":"784a5723dc251a3d6cb2d6dbaee0ba965e80a0c2127089bf35a610b636dc9f95","n":8192,"r":8,"p":1},"mac":"ac8a44c141ec6b5d4a477033bfd8adf068ee1528f1428472a717f1fef402fb16"}}
```

Comment:
This is a keystore file. Not sure you'd need it in the repo. It doesn't look like you're referencing this anywhere else.